### ISSUE TITLE
Slic_send_pings_are_called failure with Slow VMs

### ISSUE THREAD

**Issue by pepone** [icerpc/icerpc-csharp#4098]
```
  IceRpc.Tests test net8.0 failed with 1 error(s) (28.1s)
    C:\Users\jose\source\repos\icerpc-csharp\tests\IceRpc.Tests\Transports\Slic\SlicIdleTimeoutTests.cs(77): error TESTERROR:
      IceRpc:Tests:Transports:Slic:SlicIdleTimeoutTests:Slic_send_pings_are_called (717ms): Error Message:   Assert.That(TimeSpan.FromMillisecon      ds(Environment.TickCount64) - startTime, Is.LessThan(TimeSpan.FromMilliseconds(500)))
        Expected: less than 00:00:00.5000000
        But was:  00:00:00.6090000

      Stack Trace:
         at IceRpc.Tests.Transports.Slic.SlicIdleTimeoutTests.Slic_send_pings_are_called() in C:\Users\jose\source\repos\icerpc-csharp\tests\Ice      Rpc.Tests\Transports\Slic\SlicIdleTimeoutTests.cs:line 77
         at IceRpc.Tests.Transports.Slic.SlicIdleTimeoutTests.Slic_send_pings_are_called() in C:\Users\jose\source\repos\icerpc-csharp\tests\Ice      Rpc.Tests\Transports\Slic\SlicIdleTimeoutTests.cs:line 77
         at NUnit.Framework.Internal.TaskAwaitAdapter.GenericAdapter`1.BlockUntilCompleted()
         at NUnit.Framework.Internal.MessagePumpStrategy.NoMessagePumpStrategy.WaitForCompletion(AwaitAdapter awaiter)
         at NUnit.Framework.Internal.AsyncToSyncAdapter.Await[TResult](Func`1 invoke)
         at NUnit.Framework.Internal.AsyncToSyncAdapter.Await(Func`1 invoke)
         at NUnit.Framework.Internal.Commands.TestMethodCommand.RunTestMethod(TestExecutionContext context)
         at NUnit.Framework.Internal.Commands.TestMethodCommand.Execute(TestExecutionContext context)
         at NUnit.Framework.Internal.Execution.SimpleWorkItem.<>c__DisplayClass3_0.<PerformWork>b__0()
         at NUnit.Framework.Internal.ContextUtils.<>c__DisplayClass1_0`1.<DoIsolated>b__0(Object _)
```

**Comment by pepone** [comment]
I just got this once, difficult to reproduce even on a Slow VM. The failure just show that the timer took a bit more than expected `609 ms` instead of less than `500 ms` maybe we should just relax this a bit.

**Comment by pepone** [comment]
another failure 

```
  IceRpc.Tests test net9.0 failed with 1 error(s) (4.3s)
    C:\Users\jose\source\repos\icerpc-csharp\tests\IceRpc.Tests\Transports\Slic\SlicIdleTimeoutTests.cs(77): error TESTERROR:
      IceRpc:Tests:Transports:Slic:SlicIdleTimeoutTests:Slic_send_pings_are_called (806ms): Error Message:   Assert.That(TimeSpan.FromMilliseconds(Environment.TickCount64) - startTime, Is.
      LessThan(TimeSpan.FromMilliseconds(500)))
        Expected: less than 00:00:00.5000000
        But was:  00:00:00.5320000

      Stack Trace:
         at IceRpc.Tests.Transports.Slic.SlicIdleTimeoutTests.Slic_send_pings_are_called() in C:\Users\jose\source\repos\icerpc-csharp\tests\IceRpc.Tests\Transports\Slic\SlicIdleTimeoutTes
      ts.cs:line 77
         at IceRpc.Tests.Transports.Slic.SlicIdleTimeoutTests.Slic_send_pings_are_called() in C:\Users\jose\source\repos\icerpc-csharp\tests\IceRpc.Tests\Transports\Slic\SlicIdleTimeoutTes
      ts.cs:line 77
         at NUnit.Framework.Internal.TaskAwaitAdapter.GenericAdapter`1.BlockUntilCompleted()
         at NUnit.Framework.Internal.MessagePumpStrategy.NoMessagePumpStrategy.WaitForCompletion(AwaitAdapter awaiter)
         at NUnit.Framework.Internal.AsyncToSyncAdapter.Await[TResult](Func`1 invoke)
         at NUnit.Framework.Internal.AsyncToSyncAdapter.Await(Func`1 invoke)
         at NUnit.Framework.Internal.Commands.TestMethodCommand.RunTestMethod(TestExecutionContext context)
         at NUnit.Framework.Internal.Commands.TestMethodCommand.Execute(TestExecutionContext context)
         at NUnit.Framework.Internal.Execution.SimpleWorkItem.<>c__DisplayClass3_0.<PerformWork>b__0()
         at NUnit.Framework.Internal.ContextUtils.<>c__DisplayClass1_0`1.<DoIsolated>b__0(Object _)

      1)    at IceRpc.Tests.Transports.Slic.SlicIdleTimeoutTests.Slic_send_pings_are_called() in C:\Users\jose\source\repos\icerpc-csharp\tests\IceRpc.Tests\Transports\Slic\SlicIdleTimeout
      Tests.cs:line 77
  IceRpc.Tests test net8.0 succeeded (4.1s)

Test summary: total: 2, failed: 1, succeeded: 1, skipped: 0, duration: 4.3s
Build failed with 1 error(s) in 6.2s
PS C:\Users\jose\source\repos\icerpc-csharp\tests\IceRpc.Tests>
```

Running in a loop with
```
 do { dotnet test --no-build --filter FullyQualifiedName~Slic_send_pings_are_called } until ($LASTEXITCODE -ne 0)
```

**Comment by externl** [comment]
My thinking is that `[NonParallelizable]` makes the most sense for tests that are timing dependent. I think it would be better than  bumping the timeout.

**Comment by pepone** [comment]
Fixed in #4105
